R/GroupSequentialDesign.R

```diff
@@ -0,0 +1,112 @@
+#' Group-sequential design
+#'
+#' [ToDo]
+#'
+#' @slot n1 stage-one sample size
+#' @slot c1f early stopping for futility boundary
+#' @slot c1e early stopping for efficacy boundary
+#' @slot n2_continue stage-two sample size
```

Comment:
upon continuation )

Reply:
okay, to make GSDesign proper subclass of TwoStageDesign, you cannot introduce new slotnames. This is all a bit shaky since GSDesign <: TwoStageDesign: TwoStageDesigns must have same number of pivots or c2 n2 at the moment, GSDesigns not. So, GSDesign is not formally a subclass of TwoStageDesign but it's R...